app/views/ads/_ad.html.erb

```diff
@@ -1,61 +1,73 @@
+                </span>
+                Otvori oglas
+              </a>
+              <a class="dropdown-item is-flex is-align-items-center" data-clipboard-target="button" data-clipboard-text="<%= ad_url(ad.to_param) %>">
+                <span class="icon is-small mr-2">
+                  <%= inline_svg_tag('clipboard.svg', aria: true) %>
+                </span>
+                Kopiraj poveznicu
+              </a>
+            </div>
           </div>
         </div>
+      </div>
+
+      <div class="content">
+        <h3 class="is-size-5"><%= ad.zip_and_city %></h3>
+
+        <%= simple_format ad.description %>
 
         <% if ad.address.present? %>
-          <div class="control">
-            <div class="tags has-addons">
-              <span class="tag">Adresa</span>
-              <span class="tag is-info"><%= ad.address %></span>
+          <div>
+            <div class="is-flex">
+              <span class="icon mr-2">
+                <%= inline_svg_tag('location.svg', aria: true) %>
+              </span>
+              <%= ad.address %>
             </div>
-          </div>
-        <% end %>
 
-        <div class="control">
-          <a href="<%= maps_url(ad.full_address) %>" target="_blank" class="is-block">
-            <div class="tags has-addons">
-              <span class="tag">Lokacija na karti</span>
-              <span class="tag is-info">
-                <span class="icon is-small">
+            <div class="mt-2">
+              <a href="<%= maps_url(ad.full_address) %>" target="_blank" class="is-flex">
+                <span class="icon mr-2">
                   <%= inline_svg_tag('map.svg', aria: true) %>
                 </span>
-              </span>
+                Lokacija na karti
+              </a>
             </div>
-          </a>
-        </div>
-      </div>
-
-      <%= simple_format ad.description %>
-
-      <hr>
+          </div>
+        <% end %>
 
-      <div class="columns">
-        <div class="column">Telefon: <b><%= tel_to ad.phone %></b></div>
-        <div class="column">Objavljeno: <b>prije <%= time_ago_in_words ad.created_at %></b></div>
-      </div>
-      <hr>
-      <div class="columns" data-controller="clipboard">
-        <div class="column">
-          <%= link_to "Otvori oglas", ad_path(ad.to_param), class: "button" %>
-          </a>
-          <button data-clipboard-target="button" class="button" data-clipboard-text="<%= ad_url(ad.to_param) %>">
-            Kopiraj poveznicu na oglas
-          </button>
-        </div>
+        <p class="has-text-right is-size-7 mt-5"><strong>Objavljeno</strong>: prije <%= time_ago_in_words ad.created_at %></p>
       </div>
     </div>
+
+    <footer class="card-footer has-background-white-ter mt-auto">
+      <div class="card-footer-item">
+        <span class="icon mr-2">
+          <%= inline_svg_tag('phone.svg', aria: true) %>
+        </span>
+        <strong><%= tel_to ad.phone %></strong>
```

Comment:
Maybe we could even add "call" ("pozovi") around here. I see it as similar to having "lokacija na karti" (location on map"). We could make the whole address clickable (and maybe we should), but we know that not everyone using this app is tech savvy, so we could emphasize that it's possible to call immediately.

Reply:
Agree on both points